packages/cli/cli/src/commands/write-docs-definition/writeDocsDefinitionForProject.ts

```diff
@@ -22,6 +24,17 @@ export async function writeDocsDefinitionForProject({
     }
 
     await cliContext.runTaskForWorkspace(docsWorkspace, async (context) => {
+        const ossWorkspaces = (
+            await Promise.all(
+                project.apiWorkspaces.map(async (workspace) => {
+                    if (workspace instanceof OSSWorkspace) {
+                        return workspace as OSSWorkspace;
+                    }
+                    return null;
+                })
+            )
+        ).filter(isNonNullish);
```

Comment:
nit: seeing this logic in a couple places -- thoughts on making it a helper and reusing? that way if a new workspace type is introduced we'll only need to handle in one place